frontend/benefit/applicant/public/locales/en/common.json

```diff
@@ -564,6 +564,98 @@
     "home": "Helsinki benefit - Frontpage",
     "createApplication": "Helsinki benefit - Application - Create",
     "editApplication": "Helsinki benefit - Application - Edit",
-    "viewApplication": "Helsinki benefit - Application - View"
+    "viewApplication": "Helsinki benefit - Application - View",
+    "accessibilityStatement": "Helsinki benefit - Accessibility statement"
+  },
+  "accessibilityStatement": {
```

Comment:
Seems to be just basic HTML altogether. Could this part be just included from HTML file or just use `dangerouslySetInnerHTML`?

Reply:
Sure, it's just basic HTML. How about the translations then?